src/utils/string.hpp

```diff
@@ -301,6 +301,26 @@ inline std::vector<std::string> RSplit(const std::string_view src, const std::st
   return res;
 }
 
+/**
+ * Get a first part of a string by `delimiter` into a string.
+ */
```

Comment:
I think the name of the function is self-explanatory

Reply:
Do you have ideas how to name these functions better? I'm trying to find names, but currently can't create something better.